Makefile

```diff
@@ -25,25 +25,42 @@ UBUNTU_IMAGE_TAG := ubuntu:20.04
 UBUNTU_VERSION_1804 := ubuntu18.04
 PLATFORM_LINUX_ARM_64 := linux/arm64
 PLATFORM_LINUX_AMD_64 := linux/amd64
+
 ifeq "$(WITH_MPI)" "1"
-  # Don't bother supporting or building arm64+mpi builds.
-  PLATFORMS := $(PLATFORM_LINUX_AMD_64)
-  HOROVOD_WITH_MPI := 1
-  HOROVOD_WITHOUT_MPI := 0
-  HOROVOD_CPU_OPERATIONS := MPI
-  CPU_SUFFIX := -cpu-mpi
-  GPU_SUFFIX := -gpu-mpi
-  MPI_BUILD_ARG := WITH_MPI=1
+# 	Don't bother supporting or building arm64+mpi builds.
+	PLATFORMS := $(PLATFORM_LINUX_AMD_64)
+	HOROVOD_WITH_MPI := 1
+	HOROVOD_WITHOUT_MPI := 0
+	HOROVOD_CPU_OPERATIONS := MPI
+	GPU_SUFFIX := -gpu-mpi
+	MPI_BUILD_ARG := WITH_MPI=1
+
+	ifeq "$(WITH_OFI)" "1"
+		CPU_SUFFIX := -cpu-mpi-ofi
+		OFI_BUILD_ARG := WITH_OFI=1
+	else
+		CPU_SUFFIX := -cpu-mpi
+		OFI_BUILD_ARG := WITH_OFI
+	endif
 else
-  PLATFORMS := $(PLATFORM_LINUX_AMD_64),$(PLATFORM_LINUX_ARM_64)
-  WITH_MPI := 0
-  HOROVOD_WITH_MPI := 0
-  HOROVOD_WITHOUT_MPI := 1
-  HOROVOD_CPU_OPERATIONS := GLOO
-  MPI_BUILD_ARG := USE_GLOO=1
+	PLATFORMS := $(PLATFORM_LINUX_AMD_64),$(PLATFORM_LINUX_ARM_64)
+	WITH_MPI := 0
+	OFI_BUILD_ARG := WITH_OFI
+	HOROVOD_WITH_MPI := 0
+	HOROVOD_WITHOUT_MPI := 1
+	HOROVOD_CPU_OPERATIONS := GLOO
+	MPI_BUILD_ARG := USE_GLOO=1
 endif
 DEEPSPEED_VERSION := 0.7.0
 
+ifneq (,$(findstring linux/arm64,$(PLATFORMS)))
```

Comment:
Am I missing something or can the two branches of this conditional statement be moved into the previous conditional statement? I mean, `PLATFORMS` contains linux/arm64 if and only if `$(WITH_MPI)` is not equal "1".